code/modules/mob/living/silicon/ai/ai_say.dm

```diff
@@ -23,7 +23,18 @@
 	if(!radio_enabled) //AI cannot speak if radio is disabled (via intellicard) or depowered.
 		to_chat(src, span_danger("Your radio transmitter is offline!"))
 		return FALSE
-	..()
+	. = ..()
+	if(.)
+		return
```

Comment:
```suggestion
	var/parent_call = ..()
	if(parent_call)
		return parent_call
```

let's not futz with the `.` operator here because we override it in every single return option following the parent call. this is slightly cleaner. if you don't agree with the `parent_call` substitution then just keep it as `.` and put `return .` on line 28

Reply:
Went with the latter